tools/benchmarksAnalyzer/src/tests/AnalyzerTests.kt

```diff
@@ -59,7 +59,7 @@ class AnalyzerTests {
         val inputs = createMeanVarianceBenchmarks()
 
         val percent = inputs.first.calcPercentageDiff(inputs.second)
-        val expectedMean = -10.0
+        val expectedMean = -9.99809998
```

Comment:
Why?

Reply:
Because I made change to use border points of intervals for calculating, not exact means. So Result of test is changed on variance value of intervals.